.github/workflows/azure-kubernetes-service-helm.yml

```diff
@@ -109,7 +111,6 @@ jobs:
       with:
         renderEngine: 'helm'
         helmChart: ${{ env.CHART_PATH }}
-        overrideFiles: ${{ env.CHART_OVERRIDE_PATH }}
```

Comment:
Did you manage to get the override to work?

Reply:
Yes! it works! See commit "attempts override values"